### PR TITLE
Refactor couch integrity check

### PR DESCRIPTION
**Pull request by benrudolph**
- More easily define suites by predefining couchdbs
- fallback to settings for user and password
- better documentation

screen shot 2015-03-24 at 1 22 54 pm

### CODE REVIEW

**Comment by benrudolph** [comment]
wait there's an error in this

**Comment by benrudolph** [comment]
Should be good now:
screen shot 2015-03-24 at 2 14 30 pm

File: corehq/apps/cleanup/management/commands/couch_integrity.py [hunk 1]
```diff
@@ -1,61 +1,165 @@
...
+            '--wiggle',
+            dest='wiggle',
+            default=0,
+            type="int",
+            help='Define how much doc counts can be off by'
+        ),
+    )
+
     def handle_label(self, *labels, **options):
+        wiggle = options.get('wiggle', 0)
```
Comment (esoergel) [inline corehq/apps/cleanup/management/commands/couch_integrity.py]: not actionable, but I think `optparse` guarantees that the option will be available, so you can just do `options['wiggle']`

File: corehq/apps/cleanup/management/commands/couch_integrity.py [hunk 2]
```diff
@@ -1,61 +1,165 @@
...
         return
 
-    print "##### {} #####".format(view)
-
     if len(matches) == 1:
-        print u"All is consistent in each database for this view"
+        print u"All is consistent in {} for view {}".format(database, view)
         return
 
     for rows, databases in matches.items():
```
Comment (esoergel) [inline corehq/apps/cleanup/management/commands/couch_integrity.py]: Might want to call this
```python
for rows, couches in matches.items():
```
as I think the databases referred to in this block are couch hosts, not dbs with a particular couch host, right?  Seems like the nomenclature is inconsistent, which is a bit confusing.
(this is non-blocking feedback)

**Comment by czue** [comment]
looks great - thanks.

File: corehq/apps/cleanup/management/commands/couch_integrity.py (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/cleanup/management/commands/couch_integrity.py]: yes agreed, i tripped myself up a couple of times with this bad naming. will update

File: corehq/apps/cleanup/management/commands/couch_integrity.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/cleanup/management/commands/couch_integrity.py]: good point, don't know why i always feel the need to be extra safe and add a default even though there already is one

**Pull request merged**